src/blop/utils/prepare_re_env.py

```diff
@@ -1,50 +1,45 @@
 import argparse
 import datetime
 import json  # noqa F401
-
+from bluesky.callbacks.zmq import Publisher
 import bluesky.plan_stubs as bps  # noqa F401
 import bluesky.plans as bp  # noqa F401
-import databroker  # type: ignore[import-untyped]
+from tiled.client import from_uri, from_profile ##
+from bluesky.callbacks.tiled_writer import TiledWriter
 import matplotlib.pyplot as plt
 import numpy as np  # noqa F401
 from bluesky.callbacks import best_effort
 from bluesky.run_engine import RunEngine
-from databroker import Broker
-from ophyd.utils import make_dir_tree  # type: ignore[import-untyped]
+from ophyd.utils import make_dir_tree
 
 from blop.sim import HDF5Handler
 
 DEFAULT_DB_TYPE = "local"
 DEFAULT_ROOT_DIR = "/tmp/sirepo-bluesky-data"
 DEFAULT_ENV_TYPE = "stepper"
 DEFAULT_USE_SIREPO = False
+SERVER_HOST_LOCATION = "http://localhost:8000"
 
+#initializing the tiled server
+tiled_client = from_uri(SERVER_HOST_LOCATION, api_key="secret")
 
-def re_env(db_type=DEFAULT_DB_TYPE, root_dir=DEFAULT_ROOT_DIR):
-    RE = RunEngine({})
-    bec = best_effort.BestEffortCallback()
-    RE.subscribe(bec)
-
-    db = Broker.named(db_type)
-    db.reg.register_handler("HDF5", HDF5Handler, overwrite=True)
-    try:
-        databroker.assets.utils.install_sentinels(db.reg.config, version=1)
-    except Exception:
-        pass
-    RE.subscribe(db.insert)
+tiled_writer = TiledWriter(tiled_client)
+    
+def re_env(db_type="default", root_dir="/default/path"):
 
-    _ = make_dir_tree(datetime.datetime.now().year, base_path=root_dir)
+    RE = RunEngine({})   
+    bec = best_effort.BestEffortCallback()
+    RE.subscribe(bec)   
 
-    return {
-        "RE": RE,
-        "db": db,
-        "bec": bec,
-    }
+    RE.subscribe(tiled_writer) 
 
+    _ = make_dir_tree(datetime.datetime.now().year, base_path=root_dir)    
+    return dict(RE=RE, db=tiled_client, bec=bec)
```

Comment:
this can just be rewritten as:
`{RE:RE, db:tiled_client, bec:bec}`
and avoid calling dict